src/statusbar_ios.mm

```diff
@@ -56,6 +59,28 @@ static void setPreferredStatusBarStyle(UIWindow *window, UIStatusBarStyle style)
     [viewController setNeedsStatusBarAppearanceUpdate];
 }
 
+void togglePreferredStatusBarStyle()
+{
+    UIStatusBarStyle style = statusBarStyle(StatusBar::Light);
+    if(StatusBarPrivate::theme == StatusBar::Light) {
+        style = statusBarStyle(StatusBar::Dark);
+    }
+    UIWindow *keyWindow = [[UIApplication sharedApplication] keyWindow];
+    if (keyWindow)
+        setPreferredStatusBarStyle(keyWindow, style);
+
+    QTimer* timer = new QTimer();
```

Comment:
The timer object is not deleted => memory leak. Allocating it on the stack would be an option, but QTimer has also a 
handy static singleShot() method to make this more straight-forward:
```
QTimer::singleShot(200, []() {
    ...
});
```